sp800-63c/sec6_assertions.md

```diff
@@ -80,17 +80,17 @@ The assertion signature SHALL either be a digital signature using asymmetric key
 
 #### 6.2.3 <a name="encrypted-assertion"></a>Encrypted Assertion
 
-When encrypting assertions the IdP SHALL encrypt the contents of the assertion using either the RP's public key or a shared symmetric key. Shared symmetric keys used for this purpose by the IdP SHALL be independent for each RP to which they send assertions, and are normally established during registration of the RP. Public keys for encryption MAY be fetched by the IdP in a secure fashion at runtime, such as through an HTTPS URL hosted by the RP.
+When encrypting assertions, the IdP SHALL encrypt the contents of the assertion using either the RP's public key or a shared symmetric key. Shared symmetric keys used for this purpose by the IdP SHALL be independent for each RP to which they send assertions, and are normally established during registration of the RP. Public keys for encryption MAY be fetched by the IdP in a secure fashion at runtime, such as through an HTTPS URL hosted by the RP.
 
 All encryption of assertions SHALL use approved cryptography.
 
-When assertions are passed through third parties, such as a browser, the actual assertion SHALL be encrypted. For example, the XML SAML assertion using XML-Encryption. For assertions that are passed directly between IDP and RP, the actual assertion MAY be encrypted. If it is not, the assertion SHALL be sent over an authenticated protected channel.
+When assertions are passed through third parties, such as a browser, the actual assertion SHALL be encrypted. This encryption MAY include, for example, the XML SAML assertion using XML-Encryption. For assertions that are passed directly between IDP and RP, the actual assertion MAY be encrypted. If it is not, the assertion SHALL be sent over an authenticated protected channel.
```

Comment:
This is an example and should not have a normative statement attached to it in this way. Suggest rewording the sentence as:

> For example, a SAML assertion can be encrypted using XML-Encryption, or an OpenID Connect ID Token can be encrypted using JSON Web Encryption (JWE).

Not sure how I missed that we didn't have an OIDC example here, fwiw.